app/lib/ui/views/new/new_screen.dart

```diff
@@ -17,17 +17,146 @@ class NewScreen extends StatelessWidget {
+                    icon: Icon(
+                      Icons.add_circle,
+                      color: Colors.white,
+                      size: 40.0,
+                    ),
+                    color: Colors.orange,
+                    label: Text(
+                      'New Form',
+                      style: TextStyle(
+                          fontStyle: FontStyle.italic,
+                          fontSize: 20.0,
+                          fontWeight: FontWeight.bold,
+                          color: Colors.white),
+                      textAlign: TextAlign.left,
+                    ),
+                  ),
+                  Padding(
+                    padding: EdgeInsets.only(
+                      top: 10.0,
+                    ),
+                  ),
+                  RaisedButton.icon(
+                    onPressed: model.startNewAtr,
+                    padding: EdgeInsets.all(30.0),
+                    icon: Icon(
+                      Icons.wifi_protected_setup,
+                      color: Colors.white,
+                      size: 40.0,
+                    ),
+                    color: Colors.orange,
+                    label: Text(
+                      'Active Form',
+                      style: TextStyle(
+                          fontStyle: FontStyle.italic,
+                          fontSize: 20.0,
+                          fontWeight: FontWeight.bold,
+                          color: Colors.white),
+                    ),
+                  ),
+                  Padding(
+                    padding: EdgeInsets.only(
+                      top: 10.0,
+                    ),
+                  ),
+                  RaisedButton.icon(
+                    onPressed: model.startNewAtr,
+                    padding: EdgeInsets.all(30.0),
+                    icon: Icon(
+                      Icons.history_sharp,
+                      color: Colors.white,
+                      size: 40.0,
+                    ),
+                    color: Colors.orange,
+                    label: Text(
+                      'Travel History',
+                      style: TextStyle(
+                          fontStyle: FontStyle.italic,
+                          fontSize: 20.0,
+                          fontWeight: FontWeight.bold,
+                          color: Colors.white),
+                    ),
+                  ),
+                  Padding(
+                    padding: EdgeInsets.only(
+                      top: 10.0,
+                    ),
+                  ),
+                  RaisedButton.icon(
+                    onPressed: model.signOut,
+                    padding: EdgeInsets.all(30.0),
+                    icon: Icon(
+                      Icons.logout,
+                      color: Colors.white,
+                      size: 40.0,
+                    ),
+                    color: Colors.orange,
+                    label: Text(
+                      'Log Out',
```

Comment:
Can we say "Sign out" here? We call it sign out throughout the application

Reply:
> @mansiPaTeLL LGTM! Orange is fun, obviously will change later.
> 
> You can use the `NewScreenViewModel` and give it functions `navigateToActiveScreen`, etc. However, that is blocked by changes to navigator which I will be doing straight away. You can merge this as-is.

yes, orange is fun! I bored with green lol !!